sycl/doc/extensions/experimental/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -556,6 +556,98 @@ Parameters:
 
 |===
 
+==== Dynamic Command Groups
```

Comment:
This feature is complicated enough for an entry in the examples section I think (Although I know the spec is already getting too long).